### PR TITLE
Rename substitutions class names

### PR DESCRIPTION
**Pull request by lburgazzoli**

### CODE REVIEW

File: extensions/core/runtime/src/main/java/org/apache/camel/quarkus/core/runtime/graal/SubstituteIntrospectionSupport.java [hunk 1]
```diff
@@ -21,14 +21,17 @@
...
 @TargetClass(className = "org.apache.camel.support.IntrospectionSupport")
-final class Target_org_apache_camel_support_IntrospectionSupport {
-
+final class SubstituteIntrospectionSupport {
     @Alias
+    @TargetElement(name = "CACHE")
     @RecomputeFieldValue(kind = RecomputeFieldValue.Kind.FromAlias)
-    private static Map<Class<?>, BeanIntrospection.ClassInfo> CACHE = LRUCacheFactory.newLRUWeakCache(256);
-
+    private static Map<Class<?>, BeanIntrospection.ClassInfo> cache = LRUCacheFactory.newLRUWeakCache(256);
```
Comment (ppalaga) [inline extensions/core/runtime/src/main/java/org/apache/camel/quarkus/core/runtime/graal/SubstituteIntrospectionSupport.java]: Why `@TargetElement(name = "CACHE")` and `cache` field? Is our checkstyle protesting against an uppercase field name given it is not final?

Comment (lburgazzoli) [reply extensions/core/runtime/src/main/java/org/apache/camel/quarkus/core/runtime/graal/SubstituteIntrospectionSupport.java]: Yes, one of the reason for the renaming was to be able to run checkstyle against substitutions but then `final` does not make much sense for substrate and uppercase makes checkstyle unhappy so this is a compromise.

As alternative I can explicit ignore that class.

**Comment by oscerd** [comment]
LGTM, let me merge.

File: extensions/core/runtime/src/main/java/org/apache/camel/quarkus/core/runtime/graal/SubstituteIntrospectionSupport.java (hunk 1 above)
Comment (ppalaga) [reply extensions/core/runtime/src/main/java/org/apache/camel/quarkus/core/runtime/graal/SubstituteIntrospectionSupport.java]: Thanks for the explanation. BTW, there is also `// CHECKSTYLE:OFF` and `// CHECKSTYLE:ON` that can be used to skip specific lines of code.

Comment (lburgazzoli) [reply extensions/core/runtime/src/main/java/org/apache/camel/quarkus/core/runtime/graal/SubstituteIntrospectionSupport.java]: Yes, everything adds a little bit of noise, if you feel the check-style comment is better, feel free to change it

Comment (ppalaga) [reply extensions/core/runtime/src/main/java/org/apache/camel/quarkus/core/runtime/graal/SubstituteIntrospectionSupport.java]: No, your solution is one line less :-)

**Pull request merged**